src/resources/armies/moloch.json

```diff
@@ -0,0 +1,332 @@
+{
+  "id": "moloch",
+  "name": "Moloch",
+  "tokens": [
+    {
+      "id": "hq",
+      "hq": true,
+      "initiative": 0,
+      "attacks": [
+        {
+          "type": "riffle",
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
+          "angle": -1,
+          "value": 1,
+          "self": true
+        }
+      ]
+    }, {
+      "id": "blocker",
+      "count": 2,
+      "initiative": 0,
+      "health": 3,
+      "shields": [
+       0 ]
+    }, {
+      "id": "cyborg",
+      "count": 2,
+      "initiative": 3,
+      "attacks": [
+        {
+          "type": "riffle"
+        }
+      ]
+    }, {
+      "id": "gauss_cannon",
+      "initiative": 1,
+      "health": 2,
+      "attacks": [
+        {
+          "type": "cannon",
+          "damage": 1
+        }
+      ],
+      "modifiers": [
+	{
+	  "type": "nostrength",
+	  "self": true
+	}
+      ]
+    }, {
+      "id": "juggernaut",
+      "health": 2,
+      "count": 1,
+      "initiative": 1,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 2
+        }, {
+	  "type": "riffle",
+	  "angle": 1,
+	  "damage": 1
+	}
+      ],
+      "shields": [
+	0, 2, 4 
+     ]
+    }, {
+      "id": "the_clown",
+      "initiative": 2,
+      "health": 2,
+      "count": 1,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
+        }, {
+          "type": "melee",
+          "angle": 5
+        }
+      ],
+      "modifiers": [
+        {
+	  "type": "airstrike",
+	  "value": 1,
+	  "self": true
+	}
+      ]
+    }, {
+      "id": "hunter_killer",
+      "initiative": 3,
+      "health": 1,
+      "count": 2,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
+        }, {
+	  "type": "melee",
+	  "angle": 1
+	}, {
+	  "type": "melee",
+	  "angle": 3
+	}, {
+          "type": "melee",
+          "angle": 5
+        }
+      ]
+    }, {    
+      "id": "protector",
+      "initiative": 1,
+      "health": 2,
+      "count": 1,
+      "attacks": [
+        {
+          "type": "riffle",
+          "damage": 1
+        }, {
+	  "type": "riffle",
+	  "angel": 1
```

Comment:
angle

There are few others typos like this.

Reply:
All should be fixed now